libs/components/atoms/src/lib/empty-biomarker/empty-biomarker.component.ts

```diff
@@ -0,0 +1,16 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatButtonModule } from '@angular/material/button';
+
+@Component({
+  selector: 'hra-empty-biomarker',
+  standalone: true,
+  imports: [CommonModule, MatButtonModule],
+  templateUrl: './empty-biomarker.component.html',
+  styleUrls: ['./empty-biomarker.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class EmptyBiomarkerComponent {
```

Comment:
Add an input `text` that is the markdown for the text content instead of hard coding in html